_posts/2022-05-16-changelog-170.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #170"
+date: 2022-05-16 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#7849] Add inspections and quick-fixes for `unused_must_use` and `clippy::double_must_use` lints
+
+  <img src="/assets/posts/changelog-170/unused_must_use.png" width="600px"/>
+
+* [#8709] Allow installing Rust plugin together with Toml plugin without IDE restart. Note, Rust plugin update still requires IDE restart
+
+* [#5642] Add `Base URL for external documentation` option to [Advanced settings](https://www.jetbrains.com/help/idea/advanced-settings.html). This allows setting up custom base URL for external documentation, defaulting to `https://docs.rs/`  (by [@Kobzol])
+
+* [#8542] Parse generic parameters of associated types, type aliases and associated type bindings
+
+* [#8796] Enable `Emulate terminal in console` option by default in the nightly version of the plugin on macOS and Linux
```

Comment:
Do we actually need this item?
Personally, I'd drop it since it's not about release builds and we have a proper implementation in https://github.com/intellij-rust/intellij-rust/pull/8807